File: qoshimcha/task11.js
const month = 23
if (month == 12 || month ==  1 || month == 2) {
    console.log('Qish');
}else if (month >= 3 && month <= 5) {
    console.log('Bahor');
}else if (month >= 6 && month <= 8) {
    console.log('Yoz');
}else if (month >= 9 && month <= 11) {
    console.log('kuz');
}else{
    console.log('Error: wrong month number');
}

File: task11.py
user_season = int(input('Enter season: '))
if user_season == 12 or user_season == 1 or user_season == 2:
    print('Qish')
else:
    if user_season >= 3 and user_season <= 5:
        print('Bahor')
    else:
        if user_season >= 6 and user_season <= 8:
            print('Yoz')
        else:
            if user_season >= 9 and user_season <= 11:
                print('Kuz')
            else:
                print('Error: wrong moth number')